programs/stake_api/src/stake_state.rs

```diff
@@ -610,6 +608,7 @@ impl<'a> StakeAccount for KeyedAccount<'a> {
             }
             _ => return Err(InstructionError::InvalidAccountData),
         };
+        let mut to = &mut other_signers[0];
```

Comment:
I'd prefer that we rename this, or re-introduce "to" with a check to see if to.pubkey == withdrawer...

Reply:
Maybe what should be renamed is `other_signers`, since what it actually is is to_and_maybe_other_signers
The way I've written this, index 0 will be `to`, regardless of whether it is the withdrawer or not (and whether there are non-stake-account signers or not). The actual index of the withdrawer is elided above (in `authorized.check()`)

Reply:
yeah, I'm thinking that there are probably a lot of other cases here, like when an authorized staker is also the vote_account in delegate()...  Shit gets ugly fast.  I'd like for the parameters to the functions to look like real parameters, not a blob of keyed_accounts...